libs/features/mas/src/inline-price.js

```diff
@@ -288,11 +287,14 @@ export class InlinePrice extends HTMLSpanElement {
         // eslint-disable-next-line react-hooks/rules-of-hooks
         const service = getService();
         if (!service) return false;
-        const options = service.collectPriceOptions(overrides, this);
+        const priceOptions = service.collectPriceOptions(overrides, this);
+        const options = {
+          ...service.settings,
+          ...priceOptions,
+      };
         if (!options.wcsOsi.length) return false;
 
-        const ffDefaults = getParameter(FF_DEFAULTS) === 'on';
-        if (ffDefaults && (!this.dataset.displayTax || !this.dataset.forceTaxExclusive)) {
+        if (service.featureFlags[FF_DEFAULTS] && priceOptions.displayTax === undefined && priceOptions.forceTaxExclusive === undefined) {
```

Comment:
before we were checking if any of these options are absent.
but now both displayTax and forceTaxExclusive have to be absent.
is this change of logic intentional?